leader-election-api/build.gradle

```diff
@@ -4,10 +4,14 @@ libsDirName = file('build/artifacts')
 
 dependencies {
   api project(":atlasdb-commons")
+  api 'com.palantir.conjure.java:conjure-undertow-annotations'
+
   implementation project(":leader-election-api-protobufs")
+  implementation 'com.fasterxml.jackson.core:jackson-annotations'
+  implementation 'com.palantir.conjure.java:conjure-undertow-lib'
   implementation 'org.apache.commons:commons-lang3'
+  implementation 'io.undertow:undertow-core'
```

Comment:
Normally, I'd expect not needing -core and -lib. But, checkImplicitDependencies had another opinion.